pages/common/xml-validate.md

```diff
@@ -0,0 +1,24 @@
+# xml validate
+
+> Validate XML documents.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Validate XML document(s) for well-formedness only (default behavior):
+
+`xml validate --well-formed {{path/to/input1.xml|URI}} {{input2.xml ...}}`
```

Comment:
Again, could you just remove this flag?

Reply:
Yes, and (default behavior). I felt the contrast between the flags was instructive.

Reply:
- Validate XML document(s) for well-formedness only:

`xml validate {{path/to/input1.xml|URI}} {{input2.xml ...}}`